plugins/flytekit-ray/flytekitplugins/ray/models.py

```diff
@@ -178,9 +192,13 @@ def __init__(
         self,
         ray_cluster: RayCluster,
         runtime_env: typing.Optional[str],
+        ttl_seconds_after_finished: typing.Optional[int] = None,
+        shutdown_after_job_finishes: bool = False,
```

Comment:
does this mean that by default the rayjob will not be reclaimed by kuberay once the job finishes?

Reply:
According to https://github.com/ray-project/kuberay/blob/master/ray-operator/config/samples/ray_v1alpha1_rayjob.yaml#L7, Yes.